lib/arrays.md

```diff
@@ -5,44 +5,97 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+In Ruby an array is a collection of data. An array could contain multiple numbers or strings, each one having a position in the array.
```

Comment:
Yes! An array can contain another array. I suppose there's nothing stopping someone from making a matryoshka of nested arrays.